.babelrc

```diff
@@ -1,5 +1,12 @@
 {
-  "presets": [["es2015", { "modules": false }], "stage-1", "react"],
+  "presets": [
+    ["env", {
+      "modules": false,
+      "useBuiltIns": false,
```

Comment:
Does this need to be explicitly specified, since it is the default anyway?

Reply:
True. It showed what we were attempting and I made the fix from your first attempt 😄 from "usage"

Reply:
Good point, just leave it in there, then.

Reply:
Could use include and exclude:
```
["env", {
      "targets": {
        "browsers": ["last 2 versions", "safari >= 7"]
      },
      "include": ["transform-es2015-arrow-functions", "es6.map"],
      "exclude": ["transform-regenerator", "es6.set"]
    }]
```

Reply:
What would you think that would change?

Reply:
I like using a centralized browserslist in package.json.